src/PhpCsFixer/PhpCsFixerRuleSet.php

```diff
@@ -19,18 +19,34 @@
  *
  */
 
+declare(strict_types=1);
+
 namespace Centreon\PhpCsFixer;
 
 class PhpCsFixerRuleSet
 {
+    private const YEAR = 2023;
+    private const LICENSE_APACHE = 'apache';
+    private const LICENSE_PRIVATE = 'private';
+    private const LICENSES = [
```

Comment:
Some modules are missing:
- centreon-anomaly-detection
- centreon-it-edition-extensions
- centreon-map
- centreon-mbi

Reply:
Indeed, I can add them at this point but they won't be detected :
they currently don't have a `composer.json` I can rely on to be sure to detect the module.

I am not sure what is the best decision for them.